bin/install

```diff
@@ -218,4 +218,24 @@ get_ruby_type() {
 }
 
 
+install_default_gems() {
+  local default_gems="${HOME}/.default-gems"
+
+  if [ ! -f $default_gems ]; then return; fi
+
+  echo ""
+
+  for name in $(cat $default_gems); do
+    echo -n "Installing ${name} gem... "
+
+    if gem install $name > /dev/null 2>&1; then
+      echo "SUCCESS"
+    else
+      echo "FAIL"
```

Comment:
Maybe we could print the whole error message here? Or is this OK? If so, I guess the user can retry the command manually. Also, I'm not sure if we should `exit 1` if at least one of the gems fails to install.